stock_quant_analytic/tests/test_stock_quant_analytic.py

```diff
@@ -0,0 +1,109 @@
+##############################################################################
+# Copyright (c) 2023 brain-tec AG (https://braintec.com)
+# All Rights Reserved
+#
+# Licensed under the AGPL-3.0 (http://www.gnu.org/licenses/agpl.html).
+# See LICENSE file for full licensing details.
+##############################################################################
+
+from odoo.tests.common import TransactionCase
+
+
+class TestStockQuantAnalytic(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        AccountAnalyticTag = cls.env["account.analytic.tag"]
+        ProductProduct = cls.env["product.product"]
+        eur = cls.env["res.currency"].search([("name", "=", "EUR")])
+        eur.active = True
```

Comment:
What is this for?

Reply:
I thought that would fix my tests when they werent working and I forgot to remove it